services/src/test/java/org/exoplatform/onlyoffice/OnlyofficeEditorServiceTest.java

```diff
@@ -55,6 +55,63 @@ public void beforeClass() {
     this.sessionProviderService = getContainer().getComponentInstanceOfType(SessionProviderService.class);
   }
 
+  /**
+   * Test add file preferences
+   */
+  @Test
+  public void testAddFilePreferences() throws Exception {
+    // Given
+    Node node = createDocument("Test Document.docx", "nt:file", "testContent", true);
+    String[] initialMixinNodeTypes = ((NodeImpl) node).getMixinTypeNames();
+    Boolean nodePreference = node.hasNode("eoo:preferences");
+
+    // When
+    editorService.addFilePreferences(node, "john", "path");
+
+    // Then
+    assertFalse(ArrayUtils.contains(initialMixinNodeTypes, "eoo:onlyofficeFile"));
+    assertFalse(nodePreference);
+
+    String[] newMixinNodeTypes = ((NodeImpl) node).getMixinTypeNames();
+    assertTrue(ArrayUtils.contains(newMixinNodeTypes, "eoo:onlyofficeFile"));
+    assertTrue(node.getNode("eoo:preferences").hasNode("john"));
+    assertTrue(node.getNode("eoo:preferences").getNode("john").hasProperty("path"));
+    assertTrue(node.hasNode("eoo:preferences"));
```

Comment:
This assertion must be moved before the assertion:
`assertTrue(node.getNode("eoo:preferences").hasNode("john"));`
otherwise it is useless.